src/pkg/clouds/aws/secrets.go

```diff
@@ -116,18 +116,27 @@ func (a *Aws) ListSecretsByPrefix(ctx context.Context, prefix string) ([]string,
 		})
 	}
 
-	res, err := svc.DescribeParameters(ctx, &ssm.DescribeParametersInput{
-		// MaxResults: ptr.Int64(10); TODO: limit the output depending on quotas
-		ParameterFilters: filters,
-	})
-	if err != nil {
-		return nil, err
-	}
+	names := make([]string, 0, 10)
```

Comment:
By default Go will grow capacity by batches, so I think we can rely on that
```suggestion
	var names []string
```